source/install/packstack.html.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Packstack quickstart
+title: Packstack
 category: documentation
 authors: apevec, dneary, garrett, jasonbrooks, jlibosva, mmagr, pixelbeat, pmyers,
   rbowen, gbraad, cbrown2
 wiki_category: Documentation
-wiki_title: Quickstart
+wiki_title: Packstack
```

Comment:
This is historical info from wiki to website migration and it is not used anywhere afaik. @rbowen maybe we could cleanup all those wiki_* fields now?